packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -225,6 +254,49 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
       pointerSignalResolver.resolve(event);
     }
   }
+
+  void _scheduleFrameCallback() {
+    if (_frameCallbackScheduled) {
+      return;
+    }
+    final SchedulerBinding? instance = scheduler ??= SchedulerBinding.instance;
```

Comment:
Nit: I think `scheduler ??= SchedulerBinding.instance;` can be simplified a little to `scheduler ?? SchedulerBinding.instance;`

Reply:
Is 
```
final SchedulerBinding? instance = scheduler ??= SchedulerBinding.instance;
assert(instance != null);
```
equivalent as 
```
final SchedulerBinding instance = scheduler ?? SchedulerBinding.instance
```
? since the second is null-safe protected already.

Reply:
Remove from latest change that is using SchedulerBinding.instance for testing.